peekingduck/pipeline/nodes/model/yoloxv1/yolox_files/detector.py

```diff
@@ -175,17 +194,32 @@ def _load_yolox_weights(self) -> YOLOX:
         Raises:
             ValueError: `model_path` does not exist.
         """
-        if self.model_path.is_file():
-            ckpt = torch.load(str(self.model_path), map_location="cpu")
-            model = self._get_model(self.model_size).to(self.device)
-            if self.half:
-                model.half()
-            model.eval()
-            model.load_state_dict(ckpt["model"])
-
-            if self.fuse:
-                model = fuse_model(model)
+        model: Any = None
+        model_format = self.model_format
```

Comment:
(Can't add comments to the `_create_yolox_model()` section. Should we include the model format to the logging? Perhaps only for models which have multiple formats. 

Applies to MoveNet too.

Reply:
Added `model_format` in MoveNet and YOLOX logs.